tests/integration/safehtml/safehtml_test.go

```diff
@@ -23,7 +23,7 @@ func (d *dispatcher) Write(rw http.ResponseWriter, resp safehttp.Response) error
 	}
 }
 
-func (d *dispatcher) ExecuteTemplate(rw http.ResponseWriter, t safehttp.Template, data interface{}) error {
+func (dispatcher) ExecuteTemplate(rw http.ResponseWriter, t safehttp.Template, data interface{}) error {
```

Comment:
why did this need to be changed? (this is just a question)

Reply:
The `Write` and the `ExecuteTemplate` function doesn't use any fields in the dispatcher struct. So you don't need a receiver like this `(d dispatcher)` or a pointer receiver `(d *dispatcher)`. It is clearer to then remove it so you can quickly see that this function doesn't modify or need anything from the dispatcher struct.

Reply:
That makes sense, thanks!